deepface/detectors/FaceDetector.py

```diff
@@ -23,6 +23,7 @@ def build_model(detector_backend):
         "mtcnn": MtcnnWrapper.build_model,
         "retinaface": RetinaFaceWrapper.build_model,
         "mediapipe": MediapipeWrapper.build_model,
+        "yolov8n": Yolov8faceWrapper.build_model("yolov8n"),
```

Comment:
please just call build model and do not pass "yolov8n" argument. that detector will build default model.

Reply:
ok